checkstyle-sonar-plugin/src/main/java/org/sonar/plugins/checkstyle/CheckstylePlugin.java

```diff
@@ -57,6 +57,14 @@
             + "SuppressionCommentFilter</a>: "
             + FILTERS_DESCRIPTION_FOOTER;
 
+    private static final String CHECKER_TAB_WIDTH_DESCRIPTION = FILTERS_DESCRIPTION_HEADER
+            + " the <a href=\"http://checkstyle.sourceforge.net/config.html#tabWidth\">"
```

Comment:
Please use checkstyle.org in all new code.

Old places need to updated to use new url in separate issue

Reply:
Changed. I have not updated the other URLs in those decriptions as there's a separate issue for it.

Reply:
I just created #210